cmd/csender/main.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/cloudradar-monitoring/cagent/pkg/csender"
+)
+
+var (
+	// set on build:
+	// go build -o csender -ldflags="-X main.version=$(git --git-dir=src/github.com/cloudradar-monitoring/cagent/.git describe --always --long --dirty --tag)" github.com/cloudradar-monitoring/cangent/cmd/csender
+	version string
+)
+
+type boolFlag struct {
```

Comment:
Not sure, why have you created a new type for boolean CLI flags for csender? 
May be adding some comment can help.

Reply:
we have it before in csender, as we need a way to
- check if flag was intentionally set
- if it was set to true(1) or false(0)

vanilla golang's `flag.Bool` doesn't give this ability
I made a refactoring from `intFlag` we had in the` csender` to the `boolFlag` to have the value checks inside the flag itself